zulipterminal/core.py

```diff
@@ -338,6 +339,29 @@ def stream_muting_confirmation_popup(self, button: Any) -> None:
         )
         self.loop.widget = PopUpConfirmationView(self, question, mute_this_stream)
 
+    def copy_stream_email(self, stream_email: str) -> None:
+        try:
+            pyperclip.copy(stream_email)
+            clipboard_text = pyperclip.paste()
+            assert clipboard_text == stream_email
```

Comment:
It's difficult to test this since we have an external module involved which makes use of system packages (`xclip`/`xsel` on Linux, and others on different platforms). We can't know the platform on which our tests are running, and if those packages are installed in the system, nor if the text was successfully copied to clipboard.

We can test for the exception case using `side_effects` though, but I don't think that'd be any useful?

Reply:
I agree with Neil. That is what I was talking about in the first statement.

Re testing, we don't have to test the external module. We could just test whether we're calling the external module correctly.